recipes/bob.ip.facedetect/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: bob.ip.facedetect
+  version: {{ version }}
+
+source:
+  fn: bob.ip.facedetect-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.ip.facedetect/bob.ip.facedetect-{{ version }}.zip
+  md5: 98188aad12c496e026d9ebcd89a71f9b
+
+build:
+  entry_points:
+    - collect_training_data.py = bob.ip.facedetect.script.collect_training_data:main
+    - extract_training_features.py = bob.ip.facedetect.script.extract_training_features:main
+    - train_detector.py = bob.ip.facedetect.script.train_detector:main
+    - validate_detector.py = bob.ip.facedetect.script.validate_detector:main
+    - detect_faces.py = bob.ip.facedetect.script.detect_faces:main
+    - evaluate_detections.py = bob.ip.facedetect.script.evaluate:main
+    - plot_froc.py = bob.ip.facedetect.script.plot_froc:main
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.learn.boosting
+    - bob.ip.base
+    - bob.ip.draw
+    - bob.ip.color
+    - bob.io.image
+
+  run:
+    - python
+    - bob.learn.boosting
+    - bob.ip.base
+    - bob.ip.draw
+    - bob.ip.color
+    - bob.io.image
+
+test:
+  commands:
+    - collect_training_data.py --help
+    - extract_training_features.py --help
+    - train_detector.py --help
+    - validate_detector.py --help
+    - detect_faces.py --help
+    - evaluate_detections.py --help
+    - plot_froc.py --help
+    - nosetests -sv bob.ip.facedetect
+
+  imports:
+    - bob
+    - bob.ip
+    - bob.ip.facedetect
+
+  requires:
+    - nose
+
+about:
+  home: http://github.com/bioidiap/bob.ip.facedetect
+  license: Modified BSD License (3-clause)
```

Comment:
We have been allowing this one to slide actually. We were really strict for a little while, but there were some disagreements as to what qualified as an acceptable license string. This one has been allowed in practice. Though we should really pick some strict criteria and stick with them.